core/new-gui/src/app/common/typings/jquery-ui.d.ts

```diff
@@ -0,0 +1,1943 @@
+// Type definitions for jQueryUI 1.11
+// Project: http://jqueryui.com/
+// Definitions by: Boris Yankov <https://github.com/borisyankov>, John Reilly <https://github.com/johnnyreilly>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/// <reference types="jquery" />
+
+declare namespace JQueryUI {
+    // Accordion //////////////////////////////////////////////////
+
+    interface AccordionOptions extends AccordionEvents {
+        active?: any; // boolean or number
```

Comment:
I thought we weren't using JQuery. 
Again a short description about why this file is needed will be helpful.

Reply:
jquery ui is used for drag and drop later. and jquery is used by jointjs. We avoid using jquery in our own code. but when it comes to jointjs, we have to use it. Later when we introduce the drag and drop stuff, there will be description

We have to include this jquery ui type declaration file separately to let it work.